dockerscripts/docker-entrypoint.sh

```diff
@@ -35,15 +35,13 @@ docker_secrets_env_old() {
         SECRET_KEY_FILE="/run/secrets/$MINIO_SECRET_KEY_FILE"
     fi
 
-    if [ -f "$ACCESS_KEY_FILE" ] && [ -f "$SECRET_KEY_FILE" ]; then
-        if [ -f "$ACCESS_KEY_FILE" ]; then
-            MINIO_ACCESS_KEY="$(cat "$ACCESS_KEY_FILE")"
-            export MINIO_ACCESS_KEY
-        fi
-        if [ -f "$SECRET_KEY_FILE" ]; then
-            MINIO_SECRET_KEY="$(cat "$SECRET_KEY_FILE")"
-            export MINIO_SECRET_KEY
-        fi
+    if [ -f "$ACCESS_KEY_FILE" ]; then
+        MINIO_ACCESS_KEY="$(cat "$ACCESS_KEY_FILE")"
+        export MINIO_ACCESS_KEY
+    fi
+    if [ -f "$SECRET_KEY_FILE" ]; then
+        MINIO_SECRET_KEY="$(cat "$SECRET_KEY_FILE")"
+        export MINIO_SECRET_KEY
```

Comment:
I thought you wanted to make this change for MINIO_PASSWORD_FILE - while this is only done for MINIO_SECRET_FILE which is legacy and deprecated to be removed in future

Reply:
Thanks! Updated for new and old both.